### DISCUSSION TITLE
How to make Edge read-only?

### DISCUSSION THREAD

**Discussion by Luokavin** [bcakmakoglu/vue-flow#1684] [Q&A]
How to make Edge read-only, not allowing clicks, and ignore Edge when dragging on the canvas.

**Comment by allangnutzmans** [comment]
Can you provide a case?
But you can create your custom edge

`const edgeTypes: Record<string, EdgeComponent> = {
    customEdgeName: yourEdgeComponent
};`

than pass it to vueflow

`
<VueFlow
    :edge-types="edgeTypes"
    ...otherprops
`
